jerry-ext/handle-scope/handle-scope-allocator.c

```diff
@@ -126,7 +126,7 @@ jerryx_handle_scope_get_child (jerryx_handle_scope_t *scope)
   {
     return (jerryx_handle_scope_t *) jerryx_handle_scope_pool.start;
   }
-  long idx = JERRYX_HANDLE_SCOPE_PRELIST_IDX (scope);
+  intptr_t idx = JERRYX_HANDLE_SCOPE_PRELIST_IDX (scope);
```

Comment:
Shouldn't this be rather `ptrdiff_t`?

`intptr_t` is meant to hold pointers, but `JERRYX_HANDLE_SCOPE_PRELIST_IDX` computes the difference of two pointers, IIRC.

Reply:
> Shouldn't this be rather `ptrdiff_t`?
> 
> `intptr_t` is meant to hold pointers, but `JERRYX_HANDLE_SCOPE_PRELIST_IDX` computes the difference of two pointers, IIRC.

make sense